opensearch_py_ml/ml_commons/ml_commons_client.py

```diff
@@ -63,6 +63,76 @@ def upload_model(
             model_path, model_config_path, isVerbose
         )
 
+    def upload_pretrained_model(
+        self, name: str, version: str, model_format: str, load_model: bool = True
+    ):
+        """
+        This method uploads a pretrained model into opensearch cluster using ml-common plugin's api.
+        First, this method creates a model id to store model info
+        and then loads the model from Hugging Face if load_model is True.
+        The model has to be supported by ML Commons. Refer to https://opensearch.org/docs/latest/ml-commons-plugin/pretrained-models/.
+
+        :param name: Name of the pretrained model
+        :type name: string
+        :param version: Version of the pretrained model
+        :type version: string
+        :param model_format: Currently only supports "TORCH_SCRIPT"
```

Comment:
We also support `ONNX`. If you see this : https://opensearch.org/docs/latest/ml-commons-plugin/pretrained-models/, we also have urls for `ONNX` artifacts.